apps/website/src/components/EditPageLink.astro

```diff
@@ -18,9 +18,8 @@ if (page === 'components') {
 const repoUrl = 'https://github.com/iTwin/iTwinUI';
 const githubEditUrl = `${repoUrl}/edit/main/apps/website/src/pages/docs/${fileName}`;
 ---
-
 <a href={githubEditUrl} {...rest}>
-  <EditIcon aria-hidden='true' />Edit this page on GitHub
+  <EditIcon aria-hidden='true' />Edit on GitHub
```

Comment:
Maybe "Edit page on GitHub"? Currently a bit vague what this will edit

```suggestion
  <EditIcon aria-hidden='true' />Edit page on GitHub
```

Reply:
Reverted back to "Edit this page on GitHub".